visma/matrix/operations.py

```diff
@@ -220,3 +220,26 @@ def scalarDiv(const, mat):
         return matRes
     else:
         logger.error("ZeroDivisionError: Cannot divide matrix by zero")
+
+
+def traceMat(mat):
+    """Returns the trace of a square matrix (sum of diagonal elements)
+
+    Arguments:
+        mat {visma.matrix.structure.Matrix} -- matrix token
+
+    Returns:
+        trace {visma.matrix.structure.Matrix} -- 1x1 Matrix token
+    """
+    trace = Matrix()
+    trace.empty([1, 1])
+    for i in range(mat.dim[0]):
+        if len(mat.value[i][i]) != 1:
```

Comment:
Instead of this whole if-else block, you can use the following single statement.
`trace.extend(self.value[i][i])`